579: Change home page form colors and text size

kdechant opened the pull request:
Fixes #579.

I think the form field was originally white and a CSS change turned it blue. I changed it back to white and fixed a CSS font rule that was causing the font to be too small.

Tested in Firefox and Chrome on Ubuntu. The CSS for the placeholder color should work on IE/Edge as well but it hasn't been tested.

Note: I also changed the button color back to light blue as it was in commit 8d2cc7c (before the template file/navbar refactor) to make it stand out from the text field. I also tweaked some margins and padding to remove some white space that wasn't there in 8d2cc7c.

iandees commented on app/templates/index.html:
```diff
@@ -11,7 +11,7 @@ <h4>Share your ride</h4>
             <h2>{{ config.get('BRANDING_HEADLINE_2') }}</h2>
             <form method="GET" action="{{ url_for('carpool.find') }}"">
-                <input type="text" name="q" id="locationQuery" placeholder="Zip code or City, State"> <input class="btn btn-default" type="submit" value="Find a ride">
+                <input type="text" name="q" class="locationQuery" placeholder="Zip code or City, State"> <input class="btn btn-success" type="submit" value="Find a ride">
```
There is JavaScript that relies on the `id="locationQuery"` can you undo that change?

kdechant commented:
@iandees About the ID vs. class:

There were two elements with the ID #locationQuery appearing on the home page. I initially changed that to a class to make the HTML valid, though my change did break the autocomplete as you mentioned. I tried using `getElementByClassName()` instead of `getElementById()` to set up the autocomplete, but that didn't work. So I checked out commit 8d2cc7c (before the navbar refactor) and it seems that the autocomplete was working on the navbar but not on the form in the body at that point. I just removed the ID from the form field in the body, and changed the one in the navbar back to an ID. That fixes the autocomplete in the navbar, but the form in the page body still has no autocomplete. Should we create a new ticket to get that working?

iandees commented:
Autocomplete in the header only is fine for now.
